### PR TITLE
label overlapping dropdown fixed

### PR DESCRIPTION
**Pull request by vishnu552**
In edit profile label of input boxes overlapping dropdown list fixed.
Fixed dropdowns when editing a profile so it doesn't get covered by elements below.

issue #225

https://www.loom.com/share/d6d8b46234e7454881cef8aa82e50dad?sid=d94bbcb3-0704-4396-90b4-a3f530eff9b5

### CODE REVIEW

**Comment by Vayras** [comment]
@vishnu552 video looks good, lets wait for the workflows to run and start a code review for this

**Comment by Vayras** [comment]
All checks passed, Good job, will test this one locally now

**Comment by Vayras** [comment]
works as intended @elraphty @kevkevinpal I think we get this one merged

**Comment by ecurrencyhodler** [comment]
Good job @vishnu552 Let's get a maintainer to review.

**Comment by elraphty** [comment]
@vishnu552 I tried running on Chrome, this is the result

https://www.loom.com/share/93868d4c6eed4bd5ae836a273cd611f9

**Comment by vishnu552** [comment]
@elraphty sorry but, this  is not related to my work I worked on edit profile page.this is another issue #106

File: src/components/common/CreatableMultiSelect.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const S = styled(CreatableSelect)<styledProps>`
   user-select: none;
   font-size: 12px;
   border-width: 0px !important;
-
+  z-index: 900;
```
Comment (kevkevinpal) [inline src/components/common/CreatableMultiSelect.tsx:23]: this is a brittle solution what happens if we have another modal or screen that needs to be on top?

File: src/components/common/MultiSelect.tsx [hunk 2]
```diff
@@ -9,57 +9,57 @@ interface styledProps {
...
+  background: #ffffff00;
+  border: 1px solid ${(p: any) => p.color && p.color.grayish.G750};
+  color: ${(p: any) => p.color && p.color.pureBlack};
+  box-sizing: border-box;
+  box-shadow: none;
+  border: none !important;
+  user-select: none;
+  font-size: 12px;
+  border-width: 0px !important;
+  z-index: 909;
```
Comment (kevkevinpal) [inline src/components/common/MultiSelect.tsx:21]: this is a brittle solution what happens if we have another modal or screen that needs to be on top?

File: src/components/form/inputs/CreatableMultiSelectInput.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ export default function CreatableMultiSelectInput({
       >
         <R>
           <CreatableMultiSelect
-            selectStyle={{ border: 'none' }}
+            selectStyle={{ border: 'none', zIndex: '999' }}
```
Comment (kevkevinpal) [inline src/components/form/inputs/CreatableMultiSelectInput.tsx:61]: this is a brittle solution what happens if we have another modal or screen that needs to be on top?

**Review by kevkevinpal — COMMENTED** [review]
Approach NACK

I dont think using z-index is best practice and will quickly lead to the codebase being unmanageable

This article explains why z-index is not the best solution
https://medium.com/finnovate-io/why-we-never-use-z-index-231034f1bc6f

@vishnu552 there is likely an underlying reason why the dropdown is being overlapped and it might even be that z-index is being used on the thing overlapping the dropdown

**Comment by vishnu552** [comment]
@kevkevinpal I will try to find another way to resolve this problem.

**Comment by vishnu552** [comment]
@ecurrencyhodler ,I have tried different ways to resolve overlapping problem but that not giving expected result.

**Comment by ecurrencyhodler** [comment]
@vishnu552 we are going to backlog this issue. Looks like it's part of a larger issue which we are addressing here: #277 We will circle back to this issue with 277 gets resolved so I'm going to close this PR.

**Pull request closed without merging**